modules/dhcp_libvirt/dhcp_libvirt_main.rb

```diff
@@ -0,0 +1,105 @@
+require 'dhcp_libvirt/libvirt_dhcp_network'
+require 'rexml/document'
+require 'ipaddr'
+require 'dhcp_common/server'
+
+module Proxy::DHCP::Libvirt
+  class Provider < ::Proxy::DHCP::Server
+    attr_reader :libvirt_network, :network
+
+    def initialize(options = {})
+      @network = options[:network] || Proxy::DHCP::Libvirt::Plugin.settings.network
+      @libvirt_network = options[:libvirt_network] || ::Proxy::LibvirtNetwork.new(
```

Comment:
Should be ::Proxy::LibvirtDHCPNetwork. Also: I know namespaces aren't consistent in DHCP module, but I'm trying to move everything into ::Proxy::DHCP (::Proxy::DHCP::Libvirt in your case) module, could you move LibvirtDHCPNetwork into ::Proxy::DHCP::Libvirt module?